R/MzTabParam.R

```diff
@@ -0,0 +1,122 @@
+#' @title Store contents of an `XcmsExperiment` objects as a .mztabm file
+#'
+#' @name MzTabParam
+#'
+#' @export
+#'
+#' @inheritParams saveMsObject
+#'
+#' @family MS object export and import formats.
+#'
+#' @description
+#' The `saveMsObject()` and `readMsObject()` methods with the `MzTabParam`
+#' option enable users to save/load `XcmsExperiment` objects in/from a .mztabm
+#' file format. Mainly the metadata (MTD) and Small molecule feature (SMF)
```

Comment:
This might not be clear for someone who is not familiar with the mzTab-M format or xcms. I would maybe be a little more precise saying that only the `sampleData()`, `featureDefinitions()` and `featureValues()` of an `XcmsExperiment` are represented by the (or are exported to) mzTab-M file, into the *metadata* (MTD), *small molecule feature* (SMF) tables.